modules/standard/IO.chpl

```diff
@@ -4557,7 +4556,7 @@ proc channel.read(type t ...?numTypes) where numTypes > 1 {
 
 // documented in style= error= version
 pragma "no doc"
-inline proc channel.write(args ...?k, out error:syserr):bool {
+inline proc channel.write(const args ...?k, out error:syserr):bool {
```

Comment:
Are the added formal intents a way to get things working more quickly, or do you seem them as a long-term necessity?

For example, the 'const' intent on channel.write's args seems like it should be unnecessary.